### PR TITLE
https://issues.redhat.com/browse/ACM-5516 Add partial Subscription example to deploy GitOps GRC

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-5516

Just looking for a single dev review

### CODE REVIEW

**Review by JustinKuli — APPROVED** [review]
The YAML looks good to me!

**Comment by dockerymick** [comment]
@JustinKuli thanks for your approval! Going to get doc peer approval, then merge

File: governance/deploy_gitops.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [#deploying-policies-using-gitops]
 = Deploying policies using GitOps
```
Comment (oafischer) [inline governance/deploy_gitops.adoc]:
```suggestion
= Deploying policies by using GitOps
```

File: governance/deploy_gitops.adoc [hunk 2]
```diff
@@ -84,7 +84,21 @@ Your changes are committed.
...
+apiVersion: apps.open-cluster-management.io/v1
+kind: Subscription
+metadata:
+  name: subscription-example
+  namespace: sub-ns
+  annotations:
+    apps.open-cluster-management.io/git-path: sample-resources
+    apps.open-cluster-management.io/reconcile-option: replace
+spec:
+...
```
Comment (oafischer) [inline governance/deploy_gitops.adoc:100]: I like this!

**Review by oafischer — APPROVED** [review]
Looks great, one small suggestion.

**Comment by dockerymick** [comment]
@oafischer thanks so much! Since we updated the title, I updated all instances where the topic was referenced. Merging this PR

**Pull request merged**